app-arch/zstd/zstd-1.3.7-r1.ebuild

```diff
@@ -12,10 +12,10 @@ SRC_URI="https://github.com/facebook/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 LICENSE="|| ( BSD GPL-2 )"
 SLOT="0/1"
 KEYWORDS="~alpha amd64 arm arm64 hppa ia64 m68k ~mips ppc ppc64 s390 sh sparc x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
-IUSE="lz4 static-libs"
+IUSE="lz4 static-libs +threads"
 
-RDEPEND="app-arch/xz-utils
-	lz4? ( app-arch/lz4 )"
+RDEPEND="app-arch/xz-utils[threads=,${MULTILIB_USEDEP}]
```

Comment:
`-lz -llzma -llz4` is only used in linking `zstd`, which is only done on the native ABI, so the multlib USE deps are not necessary. Neither is the `threads=` USE dep. That should be evident by the fact that if `threads=` were necessary, then zstd *without* threads would not successfully link with xz-utils *with* threads.